.github/workflows/CI.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
```

Comment:
@mattdavis0351 I suppose this workflow has to change

Reply:
I tried to fix this @mattdavis0351 could you take a peek and make sure I did it right? It's in e4a03ac.

Reply:
@hectorsector I have never used the `working-directory` default, but it seems like this logic would work.